config.go

```diff
@@ -663,6 +663,12 @@ func loadConfig() (*config, error) {
 	}
 	cfg.Tor.Control = control.String()
 
+	if cfg.Tor.SOCKS == cfg.Tor.Control {
```

Comment:
Would be nice to add a commend _why_ we need to check this. Mention that wrong configuration can lead to `lnd` not starting up properly.

Reply:
Done. Thanks for the review.